src/content/docs/tutorial-service-level-mgmt/establish-capability-sli.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Establish capability SLIs"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+freshnessValidatedDate: never
+---
+
+import queriesnrqlTransactionListNrql from 'images/queries-nrql_screenshot-full_transaction-list-nrql.webp'
+
+import apmCapabilityFilterClause from 'images/apm_screenshot-crop_capability-filter-clause.webp'
+
+Having service levels on critical business capabilities, like login or authorize payment, will rapidly close the language barrier between IT and business. Service level scores on capabilities also give you better opportunities to fix service levels when they begin to degrade. For example, if the login service level begins to degrade, you'll know to look at identity mangement dependencies and workflows starting at the consumer-facing API.
```

Comment:
degrade seems a very jargon-y term. if it's well established, def use it, but if it's something only used by certain kinds of perrsonas, i might define what we mean by 'degrade'. is it 'performs worse over time than it did in a comparable time period'? e.g., we'd say a docc's performance "degrades" if the same doc loses traffic btwn Q1 and Q2?

Reply:
Thanks, I've replaced "degrade" with something less obfuscated!